client/src/menu/Anchor.tsx

```diff
@@ -286,10 +286,22 @@ export default class Anchor extends React.Component<AnchorProps, AnchorState> {
     public static isMobile(): boolean {
         return Anchor.instance.state.mobile;
     }
+
 }
 
 function CoverCard(props: { children: React.ReactNode, theme: Theme | null, onClickOutside: MouseEventHandler<HTMLDivElement> }): ReactElement {
     const ref = useRef<HTMLDivElement>(null);
+    const escFunction = useCallback((event: { keyCode: number; }) =>{
+        if(event.keyCode === 27) {

```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode says this property is deprecated.
You should probably do `event.key === 'Escape'`